content/telegraf/v1.21/introduction/getting-started.md

```diff
@@ -149,6 +149,11 @@ Notice that the timestamps occur at rounded ten second intervals (that is, `:00`
 
 That's it! You ready to use Telegraf to collect metrics and write them to your output of choice.
 
-For an introduction to Telegraf and an overview of how it works, watch the following video:
+For an introduction to Telegraf and an overview of how it works, watch the following videos:
+
+{{< youtube vGJeo3FaMds >}}
+
+
+{{< youtube k5m7SHgXRYQ >}}
+
```

Comment:
Two large videos right next to each other feels weird to me. Maybe put them in tabs?

```suggestion
For an introduction to Telegraf and an overview of how it works, watch the following videos:

{{< tabs-wrapper >}}
{{% tabs "small" %}}
[Intro to Telegraf](#)
[Telegraf Plugin Overview](#)
{{% /tabs %}}
{{% tab-content %}}
{{< youtube vGJeo3FaMds >}}
{{% /tab-content %}}
{{% tab-content %}}
{{< youtube k5m7SHgXRYQ >}}
{{% /tab-content %}}
{{< /tabs-wrapper >}}
```